src/services/config/config.service.ts

```diff
@@ -57,6 +62,23 @@ export default class ConfigService {
     });
   }
 
+  public get subgraph(): string {
+    return template(this.network.subgraph, {
+      GRAPH_KEY: this.env.GRAPH_KEY
+    });
+  }
```

Comment:
I think we could actually just get rid of the single 'subgraph' key from the configs and only have an array.